packages/jaeger-ui/src/components/common/ExternalLinks.tsx

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Dropdown, Icon, Menu } from 'antd';
+import * as React from 'react';
+import { Link } from '../../types/trace';
+
+type ExternalLinksProps = {
+  links: Link[];
+  children?: React.ReactNode;
+  className?: string;
+};
+
+const LinkValue = (props: {
+  href: string;
+  title?: string;
+  children: React.ReactNode;
+  className?: string;
+}) => (
+  <a
+    href={props.href}
+    title={props.title}
+    target="_blank"
+    rel="noopener noreferrer"
+    className={props.className}
+  >
+    {props.children} <Icon className="KeyValueTable--linkIcon" type="export" />
```

Comment:
Any update on this @rubenvp8510? I can do the change myself later this week or early next week as time permits. It should be simple since it's just for consistency's sake.

Reply:
Ups, I accidentally omitted this comment, sorry

I can do it now and update the PR